core/server/master/src/main/java/alluxio/master/file/meta/TtlBucketList.java

```diff
@@ -72,7 +80,7 @@ public int getNumBuckets() {
    * @return the total number of inodes in all the buckets
    */
   public long getNumInodes() {
-    return mNumInodes.get();
+    return mBucketList.stream().mapToInt((bucket) -> bucket.size()).sum();
```

Comment:
The size operation is constant time on the concurrent skip list set https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentSkipListSet.html#size--

Reply:
anything else u would suggest ?  since atomic counter is actually incorrect, since its a gauging metrics, i'd like it to have a chance to correct itself when gauging.

Reply:
changed the data struct